### PR TITLE
update galaxy stats dashboard

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
Copied the new JSON config for this dashboard from stats, where I used a fixed hostname for the routing panel instead of the variable, so this panel will work once again.

### CODE REVIEW

File: Galaxy.json [hunk 1]
```diff
@@ -1753,6 +1754,6 @@
   "timezone": "browser",
   "title": "Galaxy",
   "uid": "000000004",
-  "version": 60,
+  "version": 9,
```
Comment (bgruening) [inline Galaxy.json]: this looks wrong, should that be increased?

**Review by bgruening — APPROVED** [review]

File: Galaxy.json (hunk 1 above)
Comment (sanjaysrikakulam) [reply Galaxy.json]: I had the doubt as well and wasn't sure. The previous PRs I made did not change the version. So, don't know.

Comment (sanjaysrikakulam) [reply Galaxy.json]: from the docs `version of the dashboard (integer), incremented each time the dashboard is updated`

Comment (sanjaysrikakulam) [reply Galaxy.json]: I will add a commit updating the number to `61` instead.

**Comment by sanjaysrikakulam** [comment]
The dashboard is updated.

**Pull request merged**